dandi/validate.py

```diff
@@ -28,7 +28,8 @@ def validate(paths, schema_version=None, devel_debug=False):
     path, errors
       errors for a path
     """
-    for path in find_dandi_files(paths):
+    filepaths = find_files(".*", paths) if allow_any_path else find_dandi_files(paths)
```

Comment:
FWIW, I thought to suggest adding `exclude_datalad=True` but since we already default to `exclude_dotdirs = True` I guess that doesn't matter.